EIPS/eip-FOCIL.md

```diff
@@ -31,7 +31,25 @@ FOCIL is a simple committee-based design improving upon previous IL mechanisms o
 
 ### Execution Layer
 
-TBD
+On the execution layer, the block validity conditions are extended such that, after all of the transactions in the block have been executed, we attempt to execute each valid transaction in the inclusion list that was not present in the block.
```

Comment:
should we specify the validity of txns is a check on the nonce and balance for each sender?

Reply:
do we need to specify that a transaction might be invalid before any IL transactions execute but may become valid via execution of one or more transactions in the IL?